subsys/bootloader/Kconfig

```diff
@@ -134,6 +123,17 @@ config PM_PARTITION_SIZE_B0_IMAGE
 	help
 	  Flash space set aside for the B0_IMAGE partition.
 
+menuconfig IS_SECURE_BOOTLOADER
```

Comment:
On this commit in general: How do we ensure that the app doesn't set this to a different value than B0?

Reply:
Do you mean PM_PARTITION_SIZE_PROVISION? It has no prompt, so it's not modifiable.